tests/login.test.js

```diff
@@ -0,0 +1,45 @@
+import SignIn from "../pages/SignIn"
+import HomePage from "../pages/HomePage"
+import SignUp from "../pages/SignUp";
+import UserNames from "../datos/UserNames";
+
+describe('Login', () => {
+    UserNames.forEach(({username, password, reason}) => {
+    it(`Should return error when Signing In because ${reason}`, async () => {
+        await SignIn.open('bank')
+        await SignIn.logIn(username, password)
+        await expect(SignIn.signInErrorText).toBeDisplayed()
+    })
+    });
+    it('Should Sign In', async ()=> {
+        await SignIn.open('bank')
+        await SignIn.usernameInput.setValue('jsmith@demo.io')
+        await SignIn.passwordInput.setValue('Demo123!')
+        await SignIn.submit()
```

Comment:
You created a method that encapsulates the notion of logging in (`login()`). Use that instead of these 3 lines that are already included in that method